src/Microsoft.DotNet.Interactive.Formatting.Tests/FormatterTests.FormatterSources.cs

```diff
@@ -64,12 +82,35 @@ public void Convention_based_formatter_sources_are_still_registered_after_format
 
             formattedAfter.Should().Be(formattedBefore);
         }
-        
+
+        [Fact]
+        public void Convention_based_formatter_sources_can_set_preferred_MIME_type()
```

Comment:
This test seems to be a dupe of the above one. Am I missing something?

Reply:
The type being configured is different.